### PR TITLE
Accept timestamp for options.expires

### LINKED ISSUE #14: Set expiry as a length of time
99% of the times I set a cookie, the expiry is a specific amount of time from now (e.g. 1 day, 1 year) and currently need to calculate that every time.

`new Date((new Date()).getTime() + (365 * 24 * 60 * 60 * 1000))`

Would be great to have an option to pass in a length of time rather than a date object

Would either be
1/ `options.expiresIn = 365 * 24 * 60 * 60 * 1000` - But this might cause confusion as to which takes precedent if both `option.expires` and `option.expiresIn` is set

2/  if `options.expires` is an integer, treat it as a time period and add it to the current time to get the expiry date - May also cause confusion since passing different types does significantly different things

@alanclarke ?

### PR DESCRIPTION
**Pull request by KwanMan**
Resolves #14

### CODE REVIEW

**Comment by swagatha-christie** [comment]
By analyzing the blame information on this pull request, we identified @alanclarke and @davidpadbury to be potential reviewers

File: lib/cookieman.js [hunk 1]
```diff
@@ -24,7 +25,7 @@ function getCookie (name) {
 function setCookie (name, value, options) {
   var cookie = name + '=' + value + ';'
   options = options || {}
-  if (options.expires) cookie += 'Expires=' + options.expires.toUTCString() + ';'
```
Comment (alanclarke) [inline lib/cookieman.js]: i think Date accepts another date as a constuctor, so you can

```js
new Date(options.expires).toUTCString()
```

**Comment by alanclarke** [comment]
@KwanMan

**Comment by KwanMan** [comment]
TIL will udpate

**Comment by KwanMan** [comment]
@alanclarke much cleaner 😄

File: lib/cookieman.js (hunk 1 above)
Comment (activars) [inline lib/cookieman.js]: 👍

**Comment by alanclarke** [comment]
👍

**Pull request merged**